packages/benchmark/testScenarios/codeNode/codeNode.manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+	"$schema": "../testScenario.schema.json",
+	"name": "CodeNode",
+	"description": "Workflow with a single code node that is triggered using a webhook",
```

Comment:
Let's be more specific: Which code node mode? Which code node version? Which webhook version? All default settings on the webhook node?

Reply:
Good questions. This was just a placeholder for a type of scenario we might have. I remove it and add it in a separate PR